opbeat.go

```diff
@@ -161,7 +149,7 @@ func NewFromEnvironment() *Opbeat {
 	timeout := os.Getenv("OPBEAT_TIMEOUT")
 	if len(timeout) > 0 {
 		timeoutSec, err := strconv.Atoi(timeout)
-		if err != nil {
+		if err != nil && opbeat.Logger != nil {
```

Comment:
Should we create a convenience function that prints but only if it's not nill? Instead of all the not nill checks?